--- qtmultimedia/src/plugins/coreaudio/coreaudiodeviceinfo.mm.orig	2017-10-08 04:04:04.000000000 +0300
+++ qtmultimedia/src/plugins/coreaudio/coreaudiodeviceinfo.mm	2017-10-08 04:08:04.000000000 +0300
@@ -64,6 +64,10 @@
         if (CoreAudioSessionManager::instance().category() != CoreAudioSessionManager::PlayAndRecord) {
             CoreAudioSessionManager::instance().setCategory(CoreAudioSessionManager::PlayAndRecord);
         }
+    } else {
+        if (CoreAudioSessionManager::instance().category() != CoreAudioSessionManager::Playback) {
+            CoreAudioSessionManager::instance().setCategory(CoreAudioSessionManager::Playback);
+        }
     }
 #endif
 }
