--- qtmultimedia/src/plugins/coreaudio/coreaudiosessionmanager.mm.orig	2017-10-08 04:08:39.000000000 +0300
+++ qtmultimedia/src/plugins/coreaudio/coreaudiosessionmanager.mm	2017-10-08 04:12:01.000000000 +0300
@@ -216,9 +216,9 @@
 {
     m_sessionObserver = [[CoreAudioSessionObserver alloc] initWithAudioSessionManager:this];
     setActive(true);
-    // Set default category to Ambient (implies MixWithOthers). This makes sure audio stops playing
-    // if the screen is locked or if the Silent switch is toggled.
-    setCategory(CoreAudioSessionManager::Ambient, CoreAudioSessionManager::None);
+    // Set default category to Playback. This allows audio playback in background if the
+    // UIBackgroundModes audio is set in the application's info.plist.
+    setCategory(CoreAudioSessionManager::Playback, CoreAudioSessionManager::None);
 }
 
 CoreAudioSessionManager::~CoreAudioSessionManager()
